railties/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+*   Add customized prompt for Rails console
+
+    Currently, Rails console uses the default IRB prompt, which makes it easily
+    confused with normal IRB sessions and the prompt is not helpful for Rails development.
+
+    The new Rails console prompt is shorter and shows its environment:
+
+    ```txt
+    dev:001> # for develoment
+    prod:001> # for production
+    test:001> # for test
+    ```
```

Comment:
For a CHANGELOG entry, this reads a bit awkwardly to me:

> Currently, Rails console uses the default IRB prompt, which makes it easily confused with normal IRB sessions and the prompt is not helpful for Rails development.

What do you think of something like the following?

```markdown
*   Rails console now indicates the current Rails environment:

    ```txt
    dev:001> # for RAILS_ENV=development
    test:001> # for RAILS_ENV=test
    prod:001> # for RAILS_ENV=production
    my_env:001> # for RAILS_ENV=my_env
    ```

    The environment name will also be colorized when the environment is 
    `development` (green), `test` (green), or `production` (red), if your 
    terminal supports it.
```

Reply:
Applied 👍